### PR TITLE
Add: Finding Primary storage for migration

### PR DESCRIPTION
**Pull request by rajujith**
Adding a section describing how cloudstack finds a suitable primary storage for migration. 

----
:books: Documentation preview :books:: https://cloudstack-documentation--325.org.readthedocs.build/en/325/

### CODE REVIEW

**Comment by borisstoyanov** [comment]
@blueorangutan docbuild

File: source/adminguide/storage.rst [hunk 1]
```diff
@@ -562,6 +562,15 @@ be restarted.
 
 #. (KVM only) Restart the VM.
 
+Finding Primary Storage for Migration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When you click on migrate volume, CloudStack lists the available primary storage.  CloudStack uses its storage pool allocators to identify the primary storages that are available and returns a list that is suitable for the selected volume's migration.
```
Comment (DaanHoogland) [inline source/adminguide/storage.rst:567]:
```suggestion
When you click on migrate volume, CloudStack lists the available primary
storage.  CloudStack uses its storage pool allocators to identify the primary
storages that are available and returns a list that is suitable for the selected
volume's migration. 
```

File: source/adminguide/storage.rst [hunk 2]
```diff
@@ -562,6 +562,15 @@ be restarted.
 
 #. (KVM only) Restart the VM.
 
+Finding Primary Storage for Migration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When you click on migrate volume, CloudStack lists the available primary storage.  CloudStack uses its storage pool allocators to identify the primary storages that are available and returns a list that is suitable for the selected volume's migration. 
+The list also could include primary storages that are mentioned as 'Not suitable'. The criteria for which the primary storages are not suitable are:
```
Comment (DaanHoogland) [inline source/adminguide/storage.rst:568]:
```suggestion
The list also could include primary storages that are mentioned as
'Not suitable'. The criteria for which the primary storages are not suitable are: 
```

File: source/adminguide/storage.rst [hunk 3]
```diff
@@ -562,6 +562,15 @@ be restarted.
...
 
+Finding Primary Storage for Migration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When you click on migrate volume, CloudStack lists the available primary storage.  CloudStack uses its storage pool allocators to identify the primary storages that are available and returns a list that is suitable for the selected volume's migration. 
+The list also could include primary storages that are mentioned as 'Not suitable'. The criteria for which the primary storages are not suitable are: 
+	- Storage tag mismatch with the volume.
+	- Doesn't have enough capacity. 
+	- Reached its disable threshold
+	- Disabled. 
+Mismatch in the type of storage such as shared /Local.
```
Comment (DaanHoogland) [inline source/adminguide/storage.rst:573]:
```suggestion
-  Storage tag mismatch with the volume.
-  Doesn't have enough capacity. 
-  Reached its disable threshold
-  Disabled.
-  Mismatch in the type of storage such as shared /Local. 
```

**Review by weizhouapache — APPROVED** [review]
code lgtm

**Comment by shwstppr** [comment]
@rajujith can you please check the outstanding review comments?

**Review by shwstppr — APPROVED** [review]

**Pull request merged**